recipes/glslang/all/conanfile.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class GlslangConan(ConanFile):
+    name = "glslang"
+    description = "Khronos-reference front end for GLSL/ESSL, partial front " \
+                  "end for HLSL, and a SPIR-V generator."
+    license = ["BSD-3-Clause", "NVIDIA"]
+    topics = ("conan", "glslang", "glsl", "hlsl", "spirv", "spir-v", "validation", "translation")
+    homepage = "https://github.com/KhronosGroup/glslang"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    short_paths = True
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_executables": [True, False],
+        "spv_remapper": [True, False],
+        "hlsl": [True, False],
+        "enable_optimizer": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_executables": True,
+        "spv_remapper": True,
+        "hlsl": True,
+        "enable_optimizer": False
```

Comment:
Any reason why optimizer is false by default? I'd recommend having it as True.

Reply:
It requires spirv-tools, I'll turn it to` True` when ready to merge ;)

Reply:
Oh right! Cool

Reply:
option enabled ;)